src/button/index.tsx

```diff
@@ -80,7 +80,7 @@ export const Button = factory(function Button({
 			id={idBase}
 			focus={focus.shouldFocus()}
 			name={name}
-			type={type}
+			type={type as any}
```

Comment:
I'm not sure that this should be an `as any`?

Reply:
Yeah I'll change it to `as 'submit' | 'reset' | 'button'`. Should have commented on this but I was wondering what we thought about whether the `'menu'` type should be adopted into the framework level or if we it is correct to treat this as an exception considering there's no browser support for `type="menu"`.

Reply:
why does it need to be cast at all? is the `property` not typed already? regarding `menu`, is it a property we currently support? if not, we should add it to our button type typings